script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -268,7 +268,7 @@ That's backwards. If that's your goal, you need to do the opposite.
 [sha:d3315454c22a82354d634a53b5bba0504b724f06]{
 -- Page 575, Offset 10158.
 -- アレは容認してはいけないもの。
-// assuming 「アレ」 is broob's very existence and not 「逃げる」 (その理由) 
+// assuming 「アレ」 is broob's very existence and not 「逃げる」 (その理由)
```

Comment:
changes in these files were made in previous pr too?

Reply:
Yeah whichever goes in first will just win. They're just from re-exporting the entire script (cleaning comments, whitespace, etc)